pkg/proxy/ipvs/proxier.go

```diff
@@ -457,6 +442,11 @@ func NewProxier(ipt utiliptables.Interface,
 		klog.Warningf("IP Family: %s, NodePortAddresses of wrong family; %s", ipFamily, strings.Join(ips, ","))
 	}
 
+	parsedExcludeCIDRs, err := utilnet.ParseCIDRs(excludeCIDRs)
```

Comment:
I would suggest removing the validation here as we've already checked that the CIDRs are valid [here](https://github.com/kubernetes/kubernetes/blob/d6a9061cb43ab98054f0f455115d87e1b20f7971/pkg/proxy/apis/config/validation/validation.go#L304), during the validation for proxy configs.

Reply:
@hanlins  yeah, you are right. But we still need it to parse string data(excludeCIDRs) to IPNet list.

Reply:
I add comment 
https://github.com/kubernetes/kubernetes/pull/99448/files#diff-1f2043db3a45960023376d03e4dd43762d1382ad7ea94e3f26c956410b749c97R445

Reply:
I see, thanks.